src/llmq/quorums_instantsend.h

```diff
@@ -113,6 +110,9 @@ class CInstantSendManager : public CRecoveredSigsListener
     std::atomic_bool isNewInstantSendEnabled{false};
 
 public:
+    CCriticalSection cs;
+    CInstantSendDb db;
+
```

Comment:
Is this the only way to implement the test? Wallet changes can certainly be done with an extra member function.